ComposureWatch/src/Components/Card/TeamTwoCardList.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import TeamTwoCards from "./TeamTwoCards";
+
+const CardList = (props) => {
+  if (props.info.length === 0) {
```

Comment:
what would you think of naming the prop `cards` instead of `info`? I think it better describes what is in the prop